jenkins/job-configs/kubernetes-jenkins/node-e2e.yaml

```diff
@@ -119,7 +119,7 @@
         - 'cadvisor':
             repoName: 'google/cadvisor'
             gitbasedir: 'github.com/google/cadvisor'
-            dockerfilePath: 'deploy/canary/Dockerfile'
+            shell: 'make canary'
```

Comment:
Did you forget to include the Makefile in this PR? I don't see a `canary` command.